dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp

```diff
@@ -158,6 +155,61 @@ std::pair<UInt64, UInt64> MinMaxIndex::getUInt64MinMax(size_t pack_index)
     return {minmaxes->get64(pack_index * 2), minmaxes->get64(pack_index * 2 + 1)};
 }
 
+RSResult MinMaxIndex::checkNullableEqual(size_t pack_index, const Field & value, const DataTypePtr & type)
+{
+    const ColumnNullable & column_nullable = static_cast<const ColumnNullable &>(*minmaxes);
+
+    const auto * raw_type = type.get();
```

Comment:
Add an assert for !type->isNullable

Reply:
> Add an assert for !type->isNullable

Caller has checked the type with `typeid_cast<const DataTypeNullable *>` and extracted the nested type with `removeNullable(type)`.